metric_learn/constraints.py

```diff
@@ -96,6 +96,6 @@ def wrap_pairs(X, constraints):
   c = np.array(constraints[2])
   d = np.array(constraints[3])
   constraints = np.vstack((np.column_stack((a, b)), np.column_stack((c, d))))
-  y = np.vstack([np.ones((len(a), 1)), - np.ones((len(c), 1))])
+  y = np.hstack([np.ones((len(a),)), - np.ones((len(c),))])
```

Comment:
In fact we should never return a column vector but a line vector (these are the ones scikit-learn likes to work on) 
Otherwise scikit-learn's checks called by our checks will return a warning

Reply:
maybe be simpler to do:

    y = np.concatenate([np.ones_like(a), -np.ones_like(c)])

Reply:
Indeed, thanks